packages/components/src/components/drawer/drawer.lite.tsx

```diff
@@ -21,24 +22,40 @@ export default function DBDrawer(props: DBDrawerProps) {
 	const dialogContainerRef = useRef<HTMLDivElement | any>(null);
 	const state = useStore<DBDrawerState>({
 		// eslint-disable-next-line @typescript-eslint/no-explicit-any
-		handleClose: (event: any, forceClose?: boolean) => {
-			if (event.key === 'Escape') {
-				event.preventDefault();
-			}
+		handleClose: (
+			event?:
+				| ClickEvent<HTMLButtonElement | HTMLDialogElement>
+				| GeneralKeyboardEvent<HTMLDialogElement>
+				| void,
+			forceClose?: boolean
+		) => {
+			if (!event) return;
```

Comment:
This only works if handleClose is only triggered by an event. Otherwise (if called with (undefined, true) ), nothing will happen... Maybe
```suggestion
			if (!event && !forceClose) return;
			
			// callback if called programatically closed
			if (forceClose) {
				event?.stopPropagation();
				props.onClose?.(event);
				return;
			}
```

Reply:
We call the function only after an `onClick` or `onKeyDown` was triggered. So the event should be there all the time, otherwise we don't need to inform the user with `props.onClose`